core/src/main/java/org/apache/accumulo/core/util/compaction/ExternalCompactionUtil.java

```diff
@@ -108,7 +108,7 @@ public static Optional<HostAndPort> findCompactionCoordinator(ClientContext cont
       }
       return Optional.of(HostAndPort.fromString(new String(address)));
     } catch (KeeperException | InterruptedException e) {
-      throw new RuntimeException(e);
+      throw new IllegalStateException(e);
```

Comment:
Not sure if it matters - but could reassert the interrupt flag on InterruptedException.